src/dynamic.js

```diff
@@ -1,3 +1,55 @@
+/**
+ * Process action and ActivityDefinition dynamic values.
+ * @param {object} act - PlanDefinition.action or ActivityDefinition. 
+ * @param evaluateExpression - Expression to retrieve from patient results.
+ * @param {object} targetResource - FHIR resource to apply dynamic values.
+ * @returns {object} - Updated resource.
+ */
+export function processDynamicValues(act, evaluateExpression, targetResource, targetAction = null) {
+  let evaluatedValues = [];
+  if (act?.dynamicValue) {
+    // Asynchronously evaluate all dynamicValues
+    evaluatedValues = act.dynamicValue.map( (dV) => {
+      if (dV?.expression?.language != 'text/cql') {
```

Comment:
This may want to be changed to text/cql-identifer instead.  The CQP ig points to the Using CQL in FHIR IG which points back to the CQL specification.  A number of versions ago the media types were broken out from simply text/cql into text/cql - used to denote that a library is of type cql , text/cql-expression - used to express that it is a full cql expression and text/cql-identifier - used to identify an expression in a library.  

When testing the existing cervical cancer screening artifacts in a HAPI server, we found that the expressions needed to have the language be text/cql-identifier for them to get evaluated.

Reply:
Updated to text/cql-identifier. 
@mickohanlon23 note this is a breaking change. The only other change I would consider breaking in this PR is in commit [5d20ce4](https://github.com/ccsm-cds-tools/encender/pull/16/commits/5d20ce4e6a7344ff1b5c37e96c2255884556a845) which fixes CarePlan.instantiatesCanonical to be an array instead of singleton as per R4 structuredefinition.